### PR TITLE
Minor changes to GKE/AKS helm-gitops

### PR DESCRIPTION
**Pull request by NillsF**
Based on the PR review for Kind Helm GitOps, making the changes from the PR review in the AKS/GKE gitops scenario.

### CODE REVIEW

File: azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_cleanup.sh [hunk 1]
```diff
@@ -5,6 +5,13 @@
 echo "Exporting environment variables"
 export resourceGroup='<Azure Resource Group Name>'
 export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export appId='<Your Azure Service Principle name>'
+export password='<Your Azure Service Principle password>'
+export tenantId='<Your Azure tenant ID>'
+
+# Logging in to Azure using service principal
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_cleanup.sh:12]:
```suggestion
# Log in to Azure using service principal
```

File: azure_arc_k8s_jumpstart/gke/gitops/helm/az_k8sconfig_helm_cleanup.sh [hunk 2]
```diff
@@ -5,6 +5,13 @@
 echo "Exporting environment variables"
 export resourceGroup='<Azure Resource Group Name>'
 export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export appId='<Your Azure Service Principle name>'
+export password='<Your Azure Service Principle password>'
+export tenantId='<Your Azure tenant ID>'
+
+# Logging in to Azure using service principal
+echo "Log in to Azure with Service Principle & Getting AKS credentials (kubeconfig)"
+az login --service-principal --username $appId --password $password --tenant $tenantId
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/gke/gitops/helm/az_k8sconfig_helm_cleanup.sh:14]:
```suggestion
echo "Log in to Azure with Service Principle"
az login --service-principal --username $appId --password $password --tenant $tenantId
```

File: azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_cleanup.sh [hunk 3]
```diff
@@ -5,6 +5,13 @@
 echo "Exporting environment variables"
 export resourceGroup='<Azure Resource Group Name>'
 export arcClusterName='<The name of your k8s cluster as it will be shown in Azure Arc>'
+export appId='<Your Azure Service Principle name>'
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_cleanup.sh:8]:
```suggestion
export appId='<Your Azure Service Principal name>'
```

File: azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_cleanup.sh [hunk 4]
```diff
@@ -5,6 +5,13 @@
...
+export password='<Your Azure Service Principle password>'
+export tenantId='<Your Azure tenant ID>'
+
+# Login to Azure using the service principal name
+echo "Log in to Azure with Service Principle & Getting AKS credentials (kubeconfig)"
+az login --service-principal --username $appId --password $password --tenant $tenantId
```
Comment (likamrat) [inline azure_arc_k8s_jumpstart/aks/gitops/helm/az_k8sconfig_helm_cleanup.sh:15]:
```suggestion
az aks get-credentials --name $arcClusterName --resource-group $resourceGroup --overwrite-existing

```

**Review by likamrat — CHANGES_REQUESTED** [review]
- looks like *az aks get-credentials* command was removed for the AKS script (?)
- Remove AKS reference from GKE

**Comment by NillsF** [comment]
Accepted changes and changed principle --> principal in a couple more places.

**Review by likamrat — APPROVED** [review]

**Pull request merged**